README.md

```diff
@@ -110,13 +110,13 @@ const view1 = new Float32Array(
   buffer, // buffer
   0 * Float32Array.BYTES_PER_ELEMENT, // offset
   6, // length
-  3 * Float32Array.BYTES_PER_ELEMENT // stride (new!)
+  3  // stride (new!)
```

Comment:
`3` is not a multiple of `Float32Array.BYTES_PER_ELEMENT` tho. 

Did you mean to define `stride` as number of elements rather than number of bytes?

Reply:
Yep, I did mean to define `stride` as a multiple of elements, not of bytes.

Reply:
Then we need to change the definition above (line 97) which reads that stride must be a multiple of of elements